### ISSUE TITLE
Outreachy Contribution (Patience Mba)

### ISSUE THREAD

**Issue by PatChizzy** [bokeh/outreach-programs#16]
@bryevdv @pavithraes 

This issue is in reference to my task 1 submission, which can be found here https://gist.github.com/PatChizzy/01af08713a10fc83ffd329695d25d310

The outputs are added as comments and i used a ramdom sample of 5000 entries from the April 2022 dataset

**Comment by PatChizzy** [comment]
worked on #4  - env setup successful

worked on #5 https://github.com/bokeh/bokeh/pull/12950
added metadata to:
    examples/model/twin_axis.py
    examples/model/transform_jitter.py

Worked on #6 --- gist url above

**Comment by bryevdv** [comment]
@PatChizzy thanks for all the links! We are working to catch up on detailed feedback but I wanted to quickly acknowledge the the micro-tasks are satisfied!

**Comment by pavithraes** [comment]
@PatChizzy Your notebook looks great! Some quick thoughts:

- Thanks for including useful section headings, I appreciated the at-a-glance view! Some more narrative text to explain why you’re plotting certain features might be nice, but not necessary.
- It's nice to see many good plots with clear labels and titles. I'll also encourage you to experiment with interactivity, but again it's not necessary.
- In the first “Trip Distance vs. Fare Amount” plot, the two `$7000` values might be anomalies, and I think we can get a better understanding of the distributions if we remove those values?
- The 5th section heading needs to be updated to match the plot.

These are just some suggestions, please feel free to make updates as you see fit! Let us know if you'd like another review, I'm also happy to call your microtask complete. 🌻

**Comment by PatChizzy** [comment]
> @PatChizzy Your notebook looks great! Some quick thoughts:
> 
> * Thanks for including useful section headings, I appreciated the at-a-glance view! Some more narrative text to explain why you’re plotting certain features might be nice, but not necessary.
> * It's nice to see many good plots with clear labels and titles. I'll also encourage you to experiment with interactivity, but again it's not necessary.
> * In the first “Trip Distance vs. Fare Amount” plot, the two `$7000` values might be anomalies, and I think we can get a better understanding of the distributions if we remove those values?
> * The 5th section heading needs to be updated to match the plot.
> 
> These are just some suggestions, please feel free to make updates as you see fit! Let us know if you'd like another review, I'm also happy to call your microtask complete. 🌻

Hello @pavithraes thank you for your review and recommendations

**Comment by PatChizzy** [comment]
@pavithraes @bryevdv could you please help clarify?

does this optional task `[documentation task from Bokeh issues tracker]` mean to add new reported issues to docs/CHANGELOG.md?

or to document a how to guide?

i dont quite understand it but want to work on it

**Comment by bryevdv** [comment]
Hi @PatChizzy we could have been clearer! The issue tracker has a label for "docs" issues:

https://github.com/bokeh/bokeh/issues?q=is%3Aopen+is%3Aissue+label%3A%22tag%3A+component%3A+docs%22

The tsk is to look for one of those issues that you would like to work on.

**Comment by PatChizzy** [comment]
> Hi @PatChizzy we could have been clearer! The issue tracker has a label for "docs" issues:
> 
> https://github.com/bokeh/bokeh/issues?q=is%3Aopen+is%3Aissue+label%3A%22tag%3A+component%3A+docs%22
> 
> The tsk is to look for one of those issues that you would like to work on.

@bryevdv i found one i'd like to work on https://github.com/bokeh/bokeh/issues/11364

is the deliverable writing a documentation on a user guide? thats what i wasnt sure about

**Comment by bryevdv** [comment]
@PatChizzy yes for that issue the deliverable is new/expanded content in the user's guide. It will probably require developing some new example code to go along with as well. I'd suggest commenting in the issue with questions, etc. (The issue description is very short, it would be good in any case to draw out more details and requirements there)
